hokusai/production.yml

```diff
@@ -53,9 +55,9 @@ spec:
     apiVersion: extensions/v1beta1
     kind: Deployment
     name: metaphysics-web
-  minReplicas: 2
+  minReplicas: 3
   maxReplicas: 5
-  targetCPUUtilizationPercentage: 80
+  targetCPUUtilizationPercentage: 120
```

Comment:
@mzikherman also the resource request determines the target value for the pod autoscaler - so as it requested `0.5` CPU, `0.5 * 1.2 = 0.6` - thus whenever the average load across all running pods `> 0.6` it will add a new one, up to `maxReplicas`.  Whenever the average load across all pods `< 0.6` it will remove one, down to `minReplicas`

Reply:
Ah, nice! Thanks for the explanation.